driver/src/main/com/mongodb/reactivestreams/client/gridfs/GridFSDownloadPublisher.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.reactivestreams.client.gridfs;
+
+import com.mongodb.client.gridfs.model.GridFSFile;
+import org.reactivestreams.Publisher;
+
+import java.nio.ByteBuffer;
+
+/**
+ * A GridFS Publisher for downloading data from GridFS
+ *
+ * <p>Provides the {@code GridFSFile} for the file to being downloaded as well as a way to control the batchsize.</p>
+ *
+ * @since 1.13
+ */
+public interface GridFSDownloadPublisher extends Publisher<ByteBuffer> {
```

Comment:
This is essentially `GridFSDownloadStream` but now is a publisher of byte buffers.